include/BQSettingStorage.hpp

```diff
@@ -123,23 +130,24 @@ class BQSettingsStorage {
     BMS::DEV::BQ76952::Status transferSetting(bool& isComplete);
 
     /**
-     * Checks to see if the settings are stored and can be used. This includes
-     * checking to see if there are settings stored, and also checks to make
-     * sure settings are not actively being written to.
+     * Check if the settings are stored and can be used
+     *
+     * This includes checking to see if there are settings stored, and also
+     * checks to make sure settings are not actively being written to.
```

Comment:
It would be best if you had an @return because your other Boolean functions have them.